### DISCUSSION TITLE
Allow plugin/install CLI command to have an all parameter to install all available plugins

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cms#11373] [Ideas] [answered]
I use the excellent https://github.com/ostark/craft-plugin-commands Yii extension with provides an `ALL` parameter to the `./craft plugin/install` command to bulk install all plugins that are detected and present in the vendor folder. Specifying handles one by one is much slower if you wanted to do this in a bulk in a single command.

Could it be considered for the native Craft CLI command to support this?

Since Craft CMS 3.5, most of the Yii extension behaviour is now present, although you can still safely use the Yii extension and it just overwrite the core CLI commands where there is crossover.

**Comment by brandonkelly** [comment, marked as answer]
This is now resolved for Craft 4.4, via #12218. The `plugin/install`, `plugin/uninstall`, `plugin/enable`, and `plugin/disable` actions each support an `--all` option, which will cause the command to apply to all applicable plugins.

**Reply by brandonkelly** [reply]
Craft 4.4 is out!
